test/helper.go

```diff
@@ -250,3 +251,9 @@ func IsPgError(is *is.I, err error, wantCode string) {
 	is.True(ok) // expected err to be a *pgconn.PgError
 	is.Equal(pgerr.Code, wantCode)
 }
+
+func Context(t *testing.T) context.Context {
```

Comment:
One comment, perhaps check if `testing.Verbose()` or `test.Short()` to determine if verbose output is needed. 
Particularly do not enable logging when short output is requested and enable it when verbose requested.